README.md

```diff
@@ -82,11 +82,44 @@ def view_requiring_flag(request):
 def other_view(request):
     return HttpResponse('flag was not set')
 
-@flag_required('MY_FLAG', fallback_view=other_view)
+@flag_required('MY_FALLBACK_FLAG', fallback_view=other_view)
 def view_with_fallback(request):
     return HttpResponse('flag was set')
 ```
 
+For more complex use a `flag_check` decorator is provided that can be used to check for a particular value of a flag, with an optional fallback.
```

Comment:
If you use `@flag_check('MY_FLAG', False)` and the flag is not defined, what happens? Perhaps that should be documented.

Edit: on further review, I see that this will count as a match. It might be worth explicitly saying that you cannot check against a flag not being defined, e.g. `@flag_check('MY_FLAG', None)` is the same as `@flag_check('MY_FLAG', False)`.